client/src/components/pages/create-task/buttons-block/buttons-block.tsx

```diff
@@ -0,0 +1,67 @@
+import { Button, Select } from 'components/basic';
+import { Label } from '@blueprintjs/core';
+import clsx from 'clsx';
+import { ButtonClasses } from 'components/basic/button';
+import React from 'react';
+import styles from './buttons-block.module.scss';
+import './buttons-block.scss';
+import { useState } from 'react';
+import { ISelectValue } from 'components/basic/select/interface';
+import { ChangeTheme } from 'components/basic/change-theme';
+
+interface IButtonsBlockProps {}
+export const ButtonsBlock = (props: IButtonsBlockProps) => {
```

Comment:
It is not obligatory, but most components use `const ..: React.FC<IProps>`

Reply:
https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/